bcda/auth/ssas.go

```diff
@@ -96,14 +96,14 @@ func (s SSASPlugin) RevokeSystemCredentials(ssasID string) error {
 }
 
 // MakeAccessToken mints an access token for the given credentials.
-func (s SSASPlugin) MakeAccessToken(credentials Credentials) (string, error) {
-	ts, err := s.client.GetToken(client.Credentials{ClientID: credentials.ClientID, ClientSecret: credentials.ClientSecret})
+func (s SSASPlugin) MakeAccessToken(credentials Credentials) (string, string, error) {
+	ts, ei, err := s.client.GetToken(client.Credentials{ClientID: credentials.ClientID, ClientSecret: credentials.ClientSecret})
```

Comment:
let's clean up the variable naming here. I know they had 'ts' existing in codebase but to make it cleaner code let's change it tokenString and expiresIn instead of 'ts' and 'ei'.